src/app/views/App.tsx

```diff
@@ -268,6 +272,25 @@ class App extends Component<IAppProps, IAppState> {
                       {`${error.statusText} - ${error.status}`}
                     </MessageBar>
                   )}
+                  {graphExplorerMode === Mode.Complete && termsOfUse && (
+                    <MessageBar
+                      messageBarType={MessageBarType.info}
+                      isMultiline={true}
+                      onDismiss={actions.clearTermsOfUse}
+                    >
+                      <FormattedMessage id='Use the Microsoft Graph API' />
+                      <br /><br />
+                      <div>
+                        <a className={classes.links} href='https://aka.ms/msgraphtou'
+                          target='_blank'>
+                          <FormattedMessage id='Terms of use' /></a>
+                        <br />
+                        <a  className={classes.links} href='https://go.microsoft.com/fwlink/?LinkId=521839'
```

Comment:
The links will be unable to go to the localized versions of the links. 
You can check in this GE classic pull request [here](https://github.com/microsoftgraph/microsoft-graph-explorer/pull/354) for inspiration

Reply:
Thanks for the heads up! This has been fixed.